core/parachain/CMakeLists.txt

```
@@ -1,3 +1,9 @@
+add_library(thread_pool
+        thread_pool.cpp
+        )
+target_link_libraries(thread_pool
+        Boost::boost
+        )
```

To follow single format:
```suggestion
add_library(thread_pool
    thread_pool.cpp
    )
target_link_libraries(thread_pool
    Boost::boost
    )
```